scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -160,41 +189,113 @@
     (log/info "In compute cluster" name ", processing pod" pod-name ";"
               "expected:" (prepare-expected-state-dict-for-logging expected-state-dict) ","
               "existing:" (prepare-existing-state-dict-for-logging existing-state-dict))
-    ; TODO: We added an :expected/starting state to the machine, to represent when a pod is starting. We map instance.status/unknown to that state
-    ; The todo is to add in cases for [:expected/starting *] for those other states.
+    ; We should have the cross product of
+    ;      :expected/starting :expected/running :expected/completed :expected/killed :missing
```

Comment:
Should this be `:task-state/*` instead of `:expected/*` as this represents the task's internal state?

Reply:
Another one for a tie-breaker :)

Reply:
See above comment.